frontend/src/components/ButtonList.tsx

```diff
@@ -1,15 +1,21 @@
 import MainButton from './MainButton';
 import styled from 'styled-components';
+import SubButton from './SubButton';
 interface ButtonListProps {
+  buttonType: string; //'main' or 'sub'
   contentList: Array<string>;
 }
 
 //흰색 가로 버튼(MainButton)들의 목록
-const ButtonList: React.FC<ButtonListProps> = ({ contentList }) => {
+const ButtonList: React.FC<ButtonListProps> = ({ buttonType, contentList }) => {
```

Comment:
최근에는 React.FC 보다 아래와 같이 함수형으로 타입스크립트 많이 적용하는데 그 구조로 수정도 가능할까요~?
[공식문서](https://ko.react.dev/learn/typescript)도 참고차 살펴보셔요~ㅎㅎ
```suggestion
function ButtonList({ buttonType, contentList }: ButtonListProps) {
```